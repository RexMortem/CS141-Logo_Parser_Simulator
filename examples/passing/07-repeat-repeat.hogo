; Repeats can be nested in other repeats.
repeat 5 [
  repeat 3 [
    forward 10
  ] 
  right 90
]