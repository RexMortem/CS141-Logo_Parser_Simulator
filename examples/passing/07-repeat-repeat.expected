[Repeat 5 [ Repeat 3 [ GoForward 10 ], TurnRight 90 ]]